### DISCUSSION TITLE
Relation manager / Resource combination

### DISCUSSION THREAD

**Discussion by falko100** [filamentphp/filament#7249] [Help] [answered]
I have a Project model/resource with many ProjectProducts. These products should always be made / accessed from the Project page. However I don't want to use modals for creating and editing products, but full resource pages. How can I accomplish this? (Trying in V3, because we need multi-tenancy)

One more thing, can we create subresources from the View page? Create button only shows up on edit page of owner.

**Comment by danharrin** [comment]
#7168

**Reply by falko100** [reply]
Hi Dan,

Thanks for the quick response.
This works up to some point, but creating a resource from this route still requires going to the edit page of the project and the project is not automatically selected when creating a product.

Is there any way to enable the relation manager actions on the view page?

**Reply by danharrin** [reply, marked as answer]
To enable the relation manager actions on the view page, override `isReadOnly()` to return `true`.

**Reply by falko100** [reply]
Thanks so much.

V3 looks amazing so far!
